streams/streams-scala/src/test/scala/org/apache/kafka/streams/scala/StreamToTableJoinScalaIntegrationTestImplicitSerdes.scala

```diff
@@ -75,6 +75,9 @@ class StreamToTableJoinScalaIntegrationTestImplicitSerdes extends StreamToTableJ
 
     val actualClicksPerRegion: java.util.List[KeyValue[String, Long]] =
       produceNConsume(userClicksTopic, userRegionsTopic, outputTopic)
+
+    Assert.assertTrue("Expected to process some data", !actualClicksPerRegion.isEmpty)
+
```

Comment:
There was a warning that `actualClicksPerRegion` was unused. Rather than removing the variable, it seemed like making some kind of assertion on the result would be desirable.